changelogs/unreleased/5544-clayton-gonsalves-small.md

```diff
@@ -0,0 +1,4 @@
+## Fix order of Global ExtAuth and Global Ratelimit
```

Comment:
We use PR #'s in the changelog filenames, so for consistency, please rename to `5559-clayton-gonsalves-small.md`.